### PR TITLE
[W] Be less verbose in the logs.

### PR DESCRIPTION
**Pull request by Hixie**
Now that we have thousands of tests, it doesn't make sense to display a separate line for each test. The result is just megabytes of logs that you have to scrub through to find error messages.

### CODE REVIEW

**Comment by Hixie** [comment]
cc @devoncarew who added `-rexpanded` in the past
cc @tvolkert who was looking at this earlier today

**Review by devoncarew — APPROVED** [review]
removing `-rexpanded` lgtm

**Review by tvolkert — APPROVED** [review]

**Comment by Hixie** [comment]
Any idea why on Windows this isn't taking effect? Does Windows not support the unexpanded mode?

**Comment by Hixie** [comment]
Windows has a different default, it turns out.

File: packages/flutter_tools/lib/src/commands/update_packages.dart [hunk 1]
```diff
@@ -144,6 +145,7 @@ class UpdatePackagesCommand extends FlutterCommand {
           exitCode: 1,
         );
       }
+      printStatus('All pubspecs were up to date.');
       return;
     }
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/commands/update_packages.dart]: cc @jonahwilliams for review of the changes in this file

Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/commands/update_packages.dart]: LGTM

**Comment by Hixie** [comment]
cc @tvolkert PTAL.

**Review by tvolkert — APPROVED** [review]
LGTM

File: dev/bots/test.dart [hunk 2]
```diff
@@ -200,23 +191,21 @@ Future<Null> _runTests({List<String> options: const <String>[]}) async {
...
-        'fuchsia_remote_debug_protocol'), options: options);
+  await _runFlutterTest(path.join(flutterRoot, 'packages', 'flutter'));
+  await _runFlutterTest(path.join(flutterRoot, 'packages', 'flutter_localizations'));
+  await _runFlutterTest(path.join(flutterRoot, 'packages', 'flutter_driver'));
+  await _runFlutterTest(path.join(flutterRoot, 'packages', 'flutter_test'));
+  await _runFlutterTest(path.join(flutterRoot, 'packages', 'fuchsia_remote_debug_protocol'));
   await _pubRunTest(path.join(flutterRoot, 'packages', 'flutter_tools'));
   await _pubRunTest(path.join(flutterRoot, 'dev', 'bots'));
-
-  await _runAllDartTests(path.join(flutterRoot, 'dev', 'devicelab'), options: options);
```
[the remark this answers is not in the document]
Comment (Hixie) [reply dev/bots/test.dart]: done.

**Comment by Hixie** [comment]
will land on green.

**Pull request merged**